# Blockchain Creation # 

![image-20201101145901426](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101145901426.png)

- after locating the geth fold running the puppeth command intializes the process

![image-20201101150909177](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101150909177.png)

- exporting the necessary files in preparation for the initialization of the Nodes.

![image-20201101151110005](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101151110005.png)

### node 1 ###

Public address of the key:   0x1D5bf95FE4Ff9F09075d05A13b6359f90f5B8Ebc
Path of the secret key file: node1\keystore\UTC--2020-11-01T22-12-14.599084500Z--1d5bf95fe4ff9f09075d05a13b6359f90f5b8ebc

### node 2 ###

Public address of the key:   0x044320e2d33711a514868298B402283Ee922058b
Path of the secret key file: node2\keystore\UTC--2020-11-01T22-13-32.075434800Z--044320e2d33711a514868298b402283ee922058b

![image-20201101151417576](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101151417576.png)

- creating nodes one and two on our local machine

### node 3 ###

Public address of the key:   0x281BE4b92C728dE8455e9B939f0b57E5e418bBDB
Path of the secret key file: node3\keystore\UTC--2020-11-01T22-20-56.117139500Z--281be4b92c728de8455e9b939f0b57e5e418bbdb

![image-20201101152215199](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101152215199.png)

- skipping to node 3 in an attempt to circumvent failure message

![image-20201101152507914](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101152507914.png)

- local machine is a bit scant on memory 
- moving forward with steps replicated in MyCrypto wallet for practice

![image-20201101153306798](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101153306798.png)

- connecting to the private network

![image-20201101153412084](C:\Users\Derek\AppData\Roaming\Typora\typora-user-images\image-20201101153412084.png)

- failed to connect, likely due to local machines inability to mine..

